.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: ruby
 rvm:
-- '2.3.3'
-- '2.4.1'
+- 2.3.6
+- 2.4.1
```

Comment:
Shouldn't this be `2.4.2`?

Reply:
Sorry, I'm not touching 2.4 in this PR beyond what YAML dump does where it strips the superfluous quotes here.  I'll come back to other versions and branches later.